strawberryfields/gbs/plot.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Plotting and visualization
+==========================
+
+**Module name:** :mod:`strawberryfields.gbs.plot`
+
+.. currentmodule:: strawberryfields.gbs.plot
+
+This module provides functionality for visualizing GBS outputs as graphs and point processes.
+
+.. autosummary::
+    _node_coords
+    edge_coords
+    plot
+    plot_subgraph
+
+Code details
+^^^^^^^^^^^^
+"""
+from typing import Tuple, Optional
+
+import networkx as nx
+import plotly.graph_objects as go
+
+
+def _node_coords(graph: nx.Graph, l: dict) -> Tuple:
+    """ Provides the coordinates for the graph nodes when given an input graph layout.
+
+    Args:
+        graph (nx.Graph): input graph
+        l (dict): dictionary of nodes and their respective coordinates
+
+    Returns:
+         Tuple: x and y coordinates for each node
```

Comment:
```suggestion
         Dict[str, list]: lists of x and y coordinates accessed as keys of a dictionary
```
:thinking: still not sure on best wording